docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -931,3 +930,68 @@ function su_humsci_profile_update_9709() {
   \Drupal::service('module_installer')->install(['paragraphs_features']);
   \Drupal::service('module_installer')->install(['paragraphs_ee']);
 }
+
+/**
+ * Remove any existing hs_row paragraph entities.
+ */
+function su_humsci_profile_update_9710($sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $paragraph_machine_name = 'hs_row';
+
+  if (empty($sandbox['ids'])) {
+    $sandbox['ids'] = $paragraph_storage->getQuery()
+      ->condition('type', $paragraph_machine_name)
+      ->accessCheck(FALSE)
+      ->execute();
+    $sandbox['total'] = count($sandbox['ids']);
+  }
+  $paragraph_ids = array_splice($sandbox['ids'], 0, 50);
+
+  foreach ($paragraph_storage->loadMultiple($paragraph_ids) as $paragraph) {
+    $paragraph->delete();
+  }
+
+  $sandbox['#finished'] = count($sandbox['ids']) ? 1 - count($sandbox['ids']) / $sandbox['total'] : 1;
+}
+
+/**
+ * Delete hs_row paragraph type.
+ */
+function su_humsci_profile_update_9711() {
+  $paragraph_machine_name = 'hs_row';
+  // Delete the field and field storage on the paragraph.
+  $fields = \Drupal::entityTypeManager()
+    ->getStorage('field_config')
+    ->loadByProperties(['bundle' => $paragraph_machine_name]);
+
+  if ($fields) {
+    foreach ($fields as $field) {
+      /** @var Drupal\field\Entity $field */
+
+      dump($field->getName());
+
```

Comment:
```suggestion
```

Looks like a debug function made its way in.

Reply:
@joegl Oops! Thanks, it's fixed now.